app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java

```diff
@@ -318,7 +327,11 @@ private NagiosExtension convertToEbeanNagiosExtension(final models.internal.Nagi
          */
         @Override
         public PagedList<models.ebean.Alert> createAlertQuery(final AlertQuery query) {
+            if (query.getOrganization() == null) {
```

Comment:
Should it just be illegal to construct a query without an organization? I would prefer to push that invariant into the definition of a query unless there is a reason it is optional at that level.